source_type_handlers_test.go

```diff
@@ -63,3 +64,35 @@ func TestSourceTypeList(t *testing.T) {
 
 	AssertLinks(t, c.Request().RequestURI, out.Links, 100, 0)
 }
+
+func TestSourceTypeGet(t *testing.T) {
+	c, rec := request.CreateTestContext(
+		http.MethodGet,
+		"/api/sources/v3.1/source_types/1",
+		nil,
+		map[string]interface{}{
+			"tenantID": int64(1),
+		})
+
+	c.SetParamNames("id")
+	c.SetParamValues("1")
+
+	err := SourceTypeGet(c)
+	if err != nil {
+		t.Error(err)
+	}
+
+	if rec.Code != 200 {
+		t.Error("Did not return 200")
+	}
+
+	var outSrc m.SourceResponse
+	err = json.Unmarshal(rec.Body.Bytes(), &outSrc)
+	if err != nil {
+		t.Error("Failed unmarshaling output")
+	}
+
+	if *outSrc.Name != "amazon" {
```

Comment:
I think there's no need to dereference `outSrc` here, since the var isn't a pointer.